adal/build.gradle

```diff
@@ -136,10 +136,10 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.10-SNAPSHOT', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.10', changing: true)
```

Comment:
Revert this to `-SNAPSHOT` in dev to pick up nightlies

Reply:
https://github.com/AzureAD/azure-activedirectory-library-for-android/issues/1434